AT/simulator_thread/simulation_parameters.py

```diff
@@ -0,0 +1,9 @@
+window_span = 60
```

Comment:
this should also be in a config file that is not python and doesn't require a server restart

Reply:
WIP?